README.md

```diff
@@ -58,6 +58,6 @@ Check out [the documentation](https://backstage.io/docs/getting-started) on how
 
 ## License
 
-Copyright 2020-2021 © Backstage Project Authors. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our Trademark Usage page: https://www.linuxfoundation.org/trademark-usage
+Copyright 2021-2021 © Backstage Project Authors. All rights reserved. The Linux Foundation has registered trademarks and uses trademarks. For a list of trademarks of The Linux Foundation, please see our Trademark Usage page: https://www.linuxfoundation.org/trademark-usage
```

Comment:
Not sure if there is really a need to update the date, but I guess that is a question for a legal team. But I don't think that this line is correct. There might be additional lines that have the same mistake, but I don't want to review 2000+ files for it 😆

Reply:
hahah okay, feel free to close!